packages/node/src/index.js

```diff
@@ -53,9 +53,18 @@ async function runLava () {
   }
   if (shouldRunApplication) {
     // execution mode
-    const lavamoatPolicy = await loadPolicy({ debugMode, policyPath })
+    const policy = await loadPolicy({ debugMode, policyPath })
+    let lavamoatPolicy = policy
+    if (fs.existsSync(policyOverridePath)) {
+      if (debugMode) console.warn(`Merging policy-override.json into policy.json`)
+      const policyOverride = await loadPolicy({ debugMode, policyPath: policyOverridePath })
+      lavamoatPolicy = mergePolicy(policy, policyOverride)
+      // TODO: Only write if merge results in changes.
+      // Would have to make a deep equal check on whole policy, which is a waste of time.
+      // mergePolicy() should be able to do it in one pass.
+      fs.writeFileSync(policyPath, jsonStringify(lavamoatPolicy, { space: 2 }))
+    }
```

Comment:
nice!
I hadn't considered applying the override here, much more ergonomic.

this is a little more detailed then i want in this high level flow, could you break it out in to a function like `maybeUpdateOverridePolicy()`

Reply:
I was thinking of doing so when I introduce the same into lavamoat-browserify. 
This would now go into core/loadPolicy IMHO